mysql-test/r/percona_userstat_threadpool.result

```diff
@@ -0,0 +1,167 @@
+DROP TABLE IF EXISTS t1;
```

Comment:
It looks like the tests require different organisation now that some things have to be tested with the threadpool:
- split percona_userstat.test into two parts: one that does not concern with connection handling (probably it will be the old percona_userstat.test content as before this patch), and another for userstat for connection handling. Leave the former alone
- Move the second part into an include file (e.g. mysql-test/include/percona_userstat_conn_handling.inc), create two testcases percona_userstat_conn_handling.test and percona_userstat_conn_handling_threadpool.test that include it